### ISSUE TITLE
Accessibility finding: If AT user is focused on Loading Indicator focus is lost when download link loads

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#136993]
### Checklist item

WEB-413	Important updates are announced to screen readers automatically

### Severity level

3, Medium. Should be fixed in 1-3 sprints post-launch.

### Page(s)/url(s)

https://staging.va.gov/records/download-va-letters/letters

### Details

Wasn't really sure which checklist item this really corresponded to, so just went with a screen reader one.

If a user is quick enough to navigate to the loading indicator with assistive tech - when the loading is complete focus can be lost and is reset to the page, requiring the user to navigate back down (observed on desktop/Safari/VoiceOver, not a problem on JAWS/Chrome or iOS/Safari/VoiceOver).

### Reproduction steps

1. Use Safari + VoiceOver on Mac and navigate to the "Foreign Medical Program Enrollment Letter" accordion button.
2. Activate the button
3. The loading indicator appears
3. Navigate down to the "Loading your letter" loading indicator
4. When the indicator disappears, focus goes back to the body of the page, instead of staying down under the FMP accordion area

### Proposed solution or next steps
Make sure fix works with https://github.com/department-of-veterans-affairs/va.gov-team/issues/136991
